zconf.h.in

```diff
@@ -127,6 +127,9 @@ typedef void const *voidpc;
 typedef void       *voidpf;
 typedef void       *voidp;
 
+#include <stdint.h>
```

Comment:
I think perhaps zconf.h include in zlib.h.in and zlib-ng.h.in should be moved below below stdint.h include. Then you don't need this stdint.h include here.

Reply:
Moving it below both stdint.h and stdarg.h simplifies the code a little...

Reply:
I think perhaps we should make the change to zlib-ng.h.in even for consistency between the two headers.

Reply:
Can do that, as I had to adjust configure too, because the ANSI C test includes zconf.h/zconf-ng.h directly...